agent/watch_handler.go

```diff
@@ -225,3 +218,25 @@ func makeWatchPlan(logger hclog.Logger, params map[string]interface{}) (*watch.P
 	}
 	return wp, nil
 }
+
+func parseWatchArgs(args interface{}) ([]string, error) {
+	switch args := args.(type) {
+	case string:
+		return []string{args}, nil
+	case []string:
```

Comment:
Are both cases require to fix this bug?

Reply:
Ah, good question. No I think only the first case is, but I added this additional case because it seems wrong to error if the input is already the desired type.